### ISSUE TITLE
[2.4][Shibboleth][Regression] authType=dummy is being used for shibboleth accounts 

### ISSUE THREAD

**Issue by SamuAlfageme** [owncloud/client#5764]
The same happened in https://github.com/owncloud/client/issues/5408, but this time is not only after client restart, but on account creation also. 

Peeking on `owncloud.cfg`:

```
[...]
1\authType=dummy
1\serverVersion=9.1.4.2
1\url=https://<serverURL>/oc-shib
```

When `authType=dummy` the field `shibboleth_shib_user` is not appended to the configuration, so no name is shown on the account tab and the accounts are broken on client restart (because of the DummyCredentials thing)

Happened on Client 2.4.0-nightly20170509

**Comment by guruz** [comment]
You mean this always happens? Or how to reproduce it?

**Comment by ckamm** [comment]
@SamuAlfageme I'm testing some more now, but I couldn't reproduce this in my initial test. Did you skip folder conffiguration?

**Comment by ckamm** [comment]
Bisected to 4ed9edb104ddf3b5196bf7f9ae8fd4827327d48a, will fix tomorrow.

**Comment by ckamm** [comment]
@guruz I'd like to remove the whole result page. Any concerns?

**Comment by SamuAlfageme** [comment]
@ckamm -> https://github.com/owncloud/client/pull/5729#issuecomment-298852148

**Comment by ckamm** [comment]
Ah, okay.

**Comment by guruz** [comment]
I'm also OK with removing it. No strong opinions...

**Comment by SamuAlfageme** [comment]
Closing here as it's fixed, discussion to kill the page moved to #5729 I'm 👍 w/ it
